### PR TITLE
markdown Description for LaunchBehavior

### PR DESCRIPTION
**Pull request by elahehrashedi**

### CODE REVIEW

File: package.nls.json [hunk 1]
```diff
@@ -148,7 +148,7 @@
...
     "cmake-tools.configuration.cmake.launchBehavior.description": "Controls what happens with the launch terminal when you launch a target.",
-    "cmake-tools.configuration.cmake.launchBehavior.reuseTerminal.description": "The launch terminal instance is reused and the target will launch as soon as the terminal is idle.",
-    "cmake-tools.configuration.cmake.launchBehavior.breakAndReuseTerminal.description": "The launch terminal instance is reused and a break command is sent to terminate any active foreground process before launching the target.",
-    "cmake-tools.configuration.cmake.launchBehavior.newTerminal.description": "A new terminal instance is created and the target is launched in it. Existing terminals are not automatically cleaned up."
+    "cmake-tools.configuration.cmake.launchBehavior.reuseTerminal.markdownDescriptions": { "message":"The launch terminal instance is reused and the target will launch as soon as the terminal is idle.", "comment": [] },
```
Comment (bobbrow) [inline package.nls.json:151]: If it doesn't need a comment, should these be left alone (line 151 & 153)?

Comment (elahehrashedi) [reply package.nls.json:151]: They were a part of the same `markdownEnumDescriptions`.

Comment (bobbrow) [reply package.nls.json:151]: do markdown descriptions have to have a comment?

Comment (elahehrashedi) [reply package.nls.json:151]: it looks like it works without a comment (I tested it).
It didn't work if I define one in `"markdownEnumDescriptions"`  and the other two in `"enumDescriptions"`, 
but it works if I define all 3 in `"markdownEnumDescriptions"` and don't add `comment` to the other two.

**Review by bobbrow — APPROVED** [review]

**Pull request merged**